src/Integrations/Integrations/DatabaseIntegrationHelper.php

```diff
@@ -63,6 +63,10 @@ public static function propagateViaSqlComments($query, $databaseService, $mode =
         $tags = [];
 
         if ($databaseService != "") {
+            $mapping = dd_trace_env_config('DD_SERVICE_MAPPING');
```

Comment:
So I think it works but I find it weird that we need to do this here.
Line 47, we take either the `peer.service` or span service. 
IIUC, the peer.service is already remapped.
I assume the service isn't as we're doing this here. It feels weird we have to do the remapping here. 
Shouldn't the span service be the remapped one already?
If not, shouldn't we move this remapping to the span object so that we don't have to duplicate this logic here?